Makefile

```diff
@@ -19,7 +19,9 @@ translate-schemas:
 	pipenv run python -m scripts.translate_schemas
 
 translation-templates:
+	pipenv run python -m scripts.eq_translations_check
 	pipenv run python -m scripts.extract_translation_templates
 
 test-translation-templates:
+	pipenv run python -m scripts.eq_translations_check
 	pipenv run python -m scripts.extract_translation_templates --test
```

Comment:
The check should also be added to the `translate-schemas` target.

Reply:
Added, thanks!👍